iowa-a/bedrock-test/vpa.yaml

```diff
@@ -1,12 +1,25 @@
 apiVersion: autoscaling.k8s.io/v1
 kind: VerticalPodAutoscaler
 metadata:
-  name: bedrock-dev-web-vpa
-  namespace: bedrock-dev
+  name: bedrock-test-web-vpa
```

Comment:
@bkochendorfer this has been wrong for some time apparently. Should we fix it? Is it necessary?